### DISCUSSION TITLE
CDU:Entering TRU WIND/ALT in CLIMB WIND page (and following)

### DISCUSSION THREAD

**Discussion by Peter-Mary-Jo** [flybywiresim/aircraft#5412] [Help] [answered]
Hello everyone,
This is probably an idiot question: for any attempt of entering [  ]°/[  ] /[      ], I got a FORMAT ERROR message. Is there a specific format to enter those parameter.
Many thanks to all for your work on this nice project.

Happy sim flying

**Comment by emsss78** [comment]
Yes I wanted to ask the same question....

**Comment by grove2501** [comment, marked as answer]
Just 180/25/200
180= wind direction
 25 = wind speed
200= flight level (20000ft)

**Reply by Peter-Mary-Jo** [reply]
Many Thanks, it works. I didn't try to enter FL so simply
